frontend/public/components/dashboard/grid.tsx

```diff
@@ -3,16 +3,17 @@ import { Grid, GridItem } from '@patternfly/react-core';
 import { global_breakpoint_lg as breakpointLG } from '@patternfly/react-tokens';
 
 import { useRefWidth } from '../utils/ref-width-hook';
+import { DashboardCardSpan } from '@console/plugin-sdk';
 
 export enum GridPosition {
   MAIN = 'MAIN',
   LEFT = 'LEFT',
   RIGHT = 'RIGHT',
 }
 
-const mapCardsToGrid = (cards: React.ComponentType<any>[], keyPrefix: string): React.ReactNode[] =>
-  cards.map((Card, index) => (
-    <GridItem key={`${keyPrefix}-${index}`} span={12}><Card /></GridItem>
+const mapCardsToGrid = (cards: GridDashboardCard[], keyPrefix: string, ignoreCardSpan?: boolean): React.ReactNode[] =>
```

Comment:
Nit: it's better to be explicit with optional value fallbacks:

```diff
-ignoreCardSpan?: boolean
+ignoreCardSpan: boolean = false
```

Reply:
fixed